packages/bridge-controller/src/selectors.ts

```diff
@@ -60,7 +60,8 @@ export type BridgeAppState = BridgeControllerState & {
   gasFeeEstimates: GasFeeEstimates;
 } & ExchangeRateControllerState & {
     participateInMetaMetrics: boolean;
-  };
+  } & { bridgeConfig: Json };
+
```

Comment:
Can we use this as the expected type from clients instead? So we can just pass in the entire state of the RemoteFeatureFlagsController

```suggestion
  } & {
    remoteFeatureFlags: {
      bridgeConfig: Json;
    };
  };
```